criu/util.c

```diff
@@ -740,6 +740,7 @@ int is_empty_dir(int dirfd)
 
 	fdir = fdopendir(dirfd);
 	if (!fdir) {
+		pr_perror("open directory stream by fd %d failed", dirfd);
 		close_safe(&dirfd);
 		return -1;
```

Comment:
Could you please add the return values description in the comment above `is_empty_dir()` please?
Something like:
```C
/*
 * is_empty_dir will always close the FD dirfd: either implicitly
 * via closedir or explicitly in case fdopendir had failed
 * 
 * return value:
 *  < 0 : open directory stream failed
 *    0 : directory is not empty
 *    1 : directory is empty
 */
int is_empty_dir(int dirfd)
```

Reply:
@rst0git Added~